index.html

```diff
@@ -22,350 +23,279 @@
-                </svg>
-              </a>
-            </li>
-            <li id="social-media-icons-item-3">
-              <a href="#">
-                <svg width="auto" height="auto" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-                  <path d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" fill="#505F79"/>
-                </svg>
-              </a>
-            </li>
-          </ul>
+      <section id="about-myself">
+        <div id="about-myself-top">
+          <div id="about-myself-top-primary-text"><h1 class="about-myself-top-primary-text-1">About<br>Myself</h1></div>
+          <div id="about-myself-top-primary-text-2">
+            <p class="about-myself-top-primary-text-2">Hello ľm  a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          </div>
+          <div id="about-myself-top-social-media"><h5 class="social-media-p">LET'S CONNECT</h5></div>
+
+          <div id="about-myself-top-social-media-icons">
+            <ul id="social-media-icons-li-2">
+              <li id="social-media-icons-item-1">
+                <a href="https://github.com/alex1779/"><img class="social" src="css/github.svg" alt="github" width="auto" height="auto"></a>
+              </li>
+              <li id="social-media-icons-item-2">
+                <a href="https://www.linkedin.com/in/alejandro-maggioni-086678b5/"><img class="social" src="css/linkedin.svg" alt="linkedin" width="auto" height="auto"></a>
+              </li>
+              <li id="social-media-icons-item-3">
+                <a href="#"><img class="social" src="css/hand.svg" alt="hand" width="auto" height="auto"></a>
```

Comment:
Kindly modify the social icons in the about myself section to match the design template. Just as in the hero section, `Let's Connect` overlaps with the social icons.

Your design:
![image](https://user-images.githubusercontent.com/20554583/194476442-29e4fbbb-796c-4e6f-94ba-d48f6381ced2.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/194476461-01638f45-7a8f-453e-8343-58353c8a4568.png)